### PR TITLE
created an accordion style dropdown for each round

### PR DESCRIPTION
**Pull request by Meleeman01**

### CODE REVIEW

File: laravel/resources/views/pages/applications/list.blade.php [hunk 1]
```diff
@@ -8,8 +8,8 @@
     </h1>
     @foreach($rounds as $round)
         <hr>
-        <h2> {{ $round->name }} </h2>
-        <div class="scrollable">
+        <h2 class="appTitle"> {{ $round->name }} </h2>
+        <div class="scrollable hidden">
```
Comment (itsrachelfish) [inline laravel/resources/views/pages/applications/list.blade.php]: @Meleeman01 As per the requirements in the original issue #63, only rounds that are older than a year should have their applications hidden by default.

Comment (Meleeman01) [reply laravel/resources/views/pages/applications/list.blade.php]: aight I got that in there. I used Carbon's isLastYear() function and it seems to do it only if the date is from last year and not if the date is exactly 1 year. not sure which one you wanted but I can change it if you like.

File: laravel/resources/views/pages/applications/list.blade.php [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
     </h1>
     @foreach($rounds as $round)
         <hr>
-        <h2> {{ $round->name }} </h2>
-        <div class="scrollable">
+        <h2 class="appTitle"> {{ $round->name }} </h2>
+
+        <!--check to see if the round end_date is a year old from today-->
+
+        @if(Carbon::parse($round->end_date)->diffInDays(Carbon::now())==365)
```
Comment (itsrachelfish) [inline laravel/resources/views/pages/applications/list.blade.php]: Um, wouldn't this only work if the difference in days is exactly 365?

Comment (Meleeman01) [reply laravel/resources/views/pages/applications/list.blade.php]: interestingly enough it works regardless when using diffInYears(), but i think thats something that could be made more clear as well. I tested both and they work.

**Pull request merged**